yt_dlp/extractor/youtube.py

```diff
@@ -2790,8 +2874,11 @@ def _extract_comment(self, comment_renderer, parent=None):
 
         text = self._get_text(comment_renderer, 'contentText')
 
-        # note: timestamp is an estimate calculated from the current time and time_text
-        timestamp, time_text = self._extract_time_text(comment_renderer, 'publishedTimeText')
+        # Timestamp is an estimate calculated from the current time and time_text
+        # FIXME: this should probably not be enabled by default
```

Comment:
There is no way for us to extract a more accurate timestamp, correct? Then let's leave it enabled by default

Reply:
it kinda goes against our defaults of not having relative time enabled by default since it can be misleading.

but I think this is out of scope of this PR and might have issues doing that anyways